src/audio/base_fw_intel.c

```diff
@@ -88,7 +89,7 @@ int basefw_vendor_hw_config(uint32_t *data_offset, char *data)
 	tuple = tlv_next(tuple);
 	/* 2 DMIC dais */
 	value =  DT_NUM_SSP_BASE + DT_NUM_HDA_IN + DT_NUM_HDA_OUT +
-			DAI_NUM_ALH_BI_DIR_LINKS + 2;
+			IPC4_DAI_NUM_ALH_BI_DIR_LINKS + 2;
```

Comment:
DT data in the future ?

Reply:
See comment one, it seems this bi-dir-link property is a legacy one.